core/src/main/resources/hudson/tasks/LogRotator/config.jelly

```diff
@@ -41,5 +41,9 @@ THE SOFTWARE.
              description="${%if not empty, only up to this number of builds have their artifacts retained}" field="artifactNumToKeepStr">
       <f:number clazz="positive-number" min="1" step="1" />
     </f:entry>
+    <f:entry title="${%Remove last stable and successful builds}"
```

Comment:
what about these? The label seems too tied to the developer writing code instead of from a user perspective imo.

```suggestion
    <f:entry title="${%Remove last build}"
```

or 

```suggestion
    <f:entry title="${%Remove last successful build}"
```

successful is still a bit confusing as 'unstable' is successful from my reading of the code.

Reply:
Picked the shorter option, more clear to me as developer :)